matrix_70X_mini.txt

```diff
@@ -0,0 +1,22 @@
+RunMinBias2011Awf1000p0 1000.0_RunMinBias2011A+RunMinBias2011A+TIER0+SKIMD+HARVESTDfst2+ALCASPLIT/step2.root RECO
```

Comment:
I'm not sure what this file is supposed to represent.
It is probably a snapshot of the workflows from the short matrix in some given release.
Please remove.